plantdescriptionengine/pom.xml

```diff
@@ -0,0 +1,170 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
This pom does not follow the structure of the project, where there is a main pom in the root of the repository and there is a pom for every module that uses the main one as parent. (See any core system for example.) This way a lot a common dependencies can remove from this pom, only the special ones (like kalix) should remain.

The version is confusing. The PDE will release as part of Arrowhead 4.4.0, so its version should be that.
PDE currently does not provide a debian installer (by using jdeb maven plugin).

Reply:
I have updated to POM to match the core system structure, but I have not yet added a debian installer.

Reply:
I have added a debian installer for the PDE. However, I've not yet managed to install any of the new versions of the core systems using the deb installers.

Reply:
I've tested the debian installer on my machine, and it works.